src/content/index.ts

```diff
@@ -130,14 +130,26 @@ const initContentScript = () => {
                 ?.textContent
             : "";
 
-        const badge =
-          config.chatMessageAuthorBadgeSelector && isHTMLElement
-            ? (
-                chatNode.querySelector(
-                  config.chatMessageAuthorBadgeSelector
-                ) as HTMLImageElement
-              )?.src
-            : "";
+        // const badge =
+        //   config.chatMessageAuthorBadgeSelector && isHTMLElement
+        //     ? (
+        //         chatNode.querySelector(
+        //           config.chatMessageAuthorBadgeSelector
+        //         ) as HTMLImageElement
+        //       )?.src
+        //     : "";
+
+        let badges: string[] = [];
```

Comment:
```js
const badges: string[] = platform === "twitch" && config.chatMessageAuthorBadgeSelector && isHTMLElement
  ? Array.from(
      chatNode.querySelectorAll(
        config.chatMessageAuthorBadgeSelector
      ) as NodeListOf<HTMLImageElement>
    ).map((img) => img.src)
  : [];
```

Reply:
This way we avoid the if statement, and exit early in case platform isn't twitch.